pkg/config/config.go

```diff
@@ -310,6 +310,76 @@ var (
 	}
 )
 
+// GetProxies returns the proxy settings from the configuration
+func GetProxies() (*Proxy, error) {
+	if proxies := Datadog.Get("proxy"); proxies != nil {
+		proxies := &Proxy{}
+		if err := Datadog.UnmarshalKey("proxy", proxies); err != nil {
```

Comment:
Why unmarshall here? Shouldn't `loadProxyFromEnv` be called instead?

Reply:
`loadProxyFromEnv` is called at the config init and set the right values in viper from the config and the env. Because the way viper works if we set `proxy.http` it will shadow every value from the config file (like `proxy.htttps). So this function is just to initialize viper in the right way.

To get the proxy nothing has changed: we just move the code from `utils/common.go` to the config package.